README.md

```diff
@@ -1,2 +1,63 @@
 # Leaderboard
-Leaderboard list app, using webpack and ES6 modules.
+Leaderboard list app, using webpack and ES6 modules.  The projeect consume the Leaderboard API using JavaScript async and await methods.
+
+![Leaderboard](../leader.png)
+## Built With
+
+- HTML
+- JavaScript
+- CSS
+- Git
+- Webpack
+
+## Getting Started
+
+In order to run this project on your computer, please make sure that you have a web browser installed. You can clone the repo to your machine.
+
+### Prerequisites
+Node package Manager(NPM)
+To install NPM on your local machine, please visit NPM official website
+Text editor(Vscode, Atom, NotePad++, Sublime)
+
+- Click on the clone button to copy the clone
+- Open Terminal and run git clone <copied address>
+- Clone the repo
+- From you terminal run " code ." to open in your code editor.
+
+### Install
+Run the following commands to install dependencies
+
+`
+npm install
+`
+### Usage
+`
+npx hint .
+npx stylelint "**/*.{css,scss}"
+npx eslint .
+`
+
+## Authors
```

Comment:
- Please, kindly make the authors singular as you've worked on this project alone.